src/app/private/course/[courseId]/profile/feedback/page.tsx

```diff
@@ -0,0 +1,167 @@
+            <ArrowBackIcon />
+          </IconButton>
+        }
+        title="Send Feedback"
+      />
+      {feedbackSubmitted ? (
+        <div className="flex flex-col min-h-[calc(100vh-56px)] items-center text-[#49454F] justify-center">
+          <div>Your feedback is submitted!</div>
+          <div>Thank you for your feedback.</div>
+        </div>
+      ) : (
+        <>
+          <Typography sx={{ marginBlock: "12px", fontWeight: 500 }}>
+            How likely are you to recommend OfficeMinutes to a friend?
+          </Typography>
+          <FormControl
+            component="fieldset"
+            sx={{ width: "100%", paddingInline: "16px" }}
+          >
+            <RadioGroup value={recommendation} onChange={handleRecommendChange}>
+              <FormControlLabel
+                value={1}
+                control={<Radio />}
+                label="Very unlikely"
+                sx={formControlLabelStyles}
+              />
+              <FormControlLabel
+                value={2}
+                control={<Radio />}
+                label="Unlikely"
+                sx={formControlLabelStyles}
+              />
+              <FormControlLabel
+                value={3}
+                control={<Radio />}
+                label="Neutral"
+                sx={formControlLabelStyles}
+              />
+              <FormControlLabel
+                value={4}
+                control={<Radio />}
+                label="Likely"
+                sx={formControlLabelStyles}
+              />
+              <FormControlLabel
+                value={5}
+                control={<Radio />}
+                label="Very likely"
+                sx={formControlLabelStyles}
+              />
+            </RadioGroup>
+          </FormControl>
+          <Typography sx={{ marginBlock: "12px", fontWeight: 500 }}>
+            How can we improve OfficeMinutes for you?
+          </Typography>
+          <TextField
+            label="Feedback"
+            variant="outlined"
+            fullWidth
+            multiline
+            rows={4}
+            value={feedback}
+            onChange={handleFeedbackChange}
+          />
+          <Button
+            sx={{
+              backgroundColor: theme.palette.primary.main,
+              color: "white",
+              textTransform: "none",
+              borderRadius: "100px",
+              marginTop: "16px",
+              paddingInline: "24px",
+              paddingBlock: "10px",
+              fontWeight: 550,
+              alignSelf: "center",
+              "&.Mui-disabled": {
+                backgroundColor: theme.palette.action.disabledBackground,
+                color: theme.palette.text.disabled,
+              },
+            }}
+            onClick={handleSendFeedback}
+            disabled={recommendation === null}
+          >
+            Send feedback
+          </Button>
+          {error !== "" ? (
```

Comment:
I'm forgetting, is `error !== && <Component />` a valid syntax?

Reply:
yeah it's valid. I think `!error && <Component />` also evaluates as well